crates/typst/src/export/svg.rs

```diff
@@ -76,6 +82,8 @@ struct SVGRenderer {
     /// The `Ratio` is the aspect ratio of the gradient, this is used to correct
     /// the angle of the gradient.
     gradients: Deduplicator<(Gradient, Ratio)>,
+    /// These are the gradients that compose a conic gradient.
+    subgradients: Deduplicator<SVGSubGradient>,
```

Comment:
Should the name of the field and type perhaps incorporate that this is for conic gradients only? Or is it not?